com.tracktion.Waveform13.yml

```diff
@@ -0,0 +1,160 @@
+id: com.tracktion.Waveform13
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+tags:
+  - proprietary
+command: waveform
+finish-args:
+  - --device=all
+  - --env=VST3_PATH=/app/extensions/Plugins/vst3
+  - --env=LADSPA_PATH=/app/extensions/Plugins/ladspa
+  - --filesystem=home/.lxvst:create
+  - --filesystem=home/.vst:create
+  - --filesystem=home/.vst3:create
+  - --filesystem=home/Tracktion
+  - --filesystem=xdg-config
```

Comment:
why would you need xdg-config?

Reply:
That must have been an oversight. I forked this from the original Waveform repo. I will remove it promptly and look over the permissions once again.